examples/ads.amp.html

```diff
@@ -719,6 +720,13 @@ <h2>Bringhub Mini-Storefront</h2>
       heights="(max-width: 270px) 1280px, (max-width:553px) 640px, 338px">
   </amp-embed>
 
+  <h2>CA A.J.A. </h2>
+  <amp-ad width="300" height="250"
```

Comment:
So it seems like you missed the attribute `widgetsIds` 😄 

<img width="1440" alt="screen shot 2019-02-04 at 12 10 05 pm" src="https://user-images.githubusercontent.com/1448289/52234184-e9c81780-2875-11e9-845c-fa3e0ddb4439.png">

Other than that, everything looks great! 👍 Thanks for working with us and implementing all the changes!